src/main/java/com/pon/assignment8/StaffController.java

```diff
@@ -0,0 +1,28 @@
+package com.pon.assignment8;
+
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+@RestController
+public class StaffController {
+        private final StaffMapper staffMapper;
+
+        public StaffController(StaffMapper staffMapper) {
+            this.staffMapper = staffMapper;
+        }
+        @GetMapping("/staff")
+        public List<Staff> findAll() {
+            return staffMapper.findAll();
+        }
+        @GetMapping("/staff/room_type")
+        public List<Staff> isSmoking(@RequestParam String smoking) {
+            return staffMapper.isSmoking(Boolean.parseBoolean(smoking));
+        }
+        @GetMapping("/companies")
+        public List<Company> findAll2() {
```

Comment:
小出しになっちゃってすみません！
Mapperのメソッドも同様にですが、メソッドの命名を具体的にしましょう！
こういった数字をマジックナンバーと呼びます。
```suggestion
        public List<Company> findAllCompany() {
```

Reply:
私にしか分からない＆特に意味のない数字ですね、staffの方も直します！